spec/components/intervention_spec.rb

```diff
@@ -8,7 +8,13 @@ def component_name
   it "renders the component" do
     render_component({})
     assert_select ".gem-c-intervention__title", text: "Check the next steps for your limited company"
-    assert_select ".gem-c-intervention__paragraph", text: "You might be interested in this because you’ve been browsing guidance relevant to starting a limited company."
-    assert_select ".gem-c-intervention__paragraph", text: "Hide this suggestion if it’s not relevant to you"
+    assert_select ".govuk-body", text: "You might be interested in this because you’ve been browsing guidance relevant to starting a limited company."
+    assert_select ".govuk-body a", text: "Hide this suggestion"
+  end
+
+  it "renders the tracking attributes" do
+    render_component({})
+    assert_select ".gem-c-intervention__title a[data-track-category='interventionBanner'][data-track-action='interventionClicked'][data-track-label='/next-steps-for-your-business'][data-track-dimension-index='29'][data-track-dimension='Check the next steps for your limited company']"
+    assert_select ".govuk-body a[data-track-category='interventionBanner'][data-track-action='interventionDismissed'][data-track-label='?hide-intervention=true'][data-track-dimension-index='29'][data-track-dimension='Hide this suggestion']"
```

Comment:
I'm not sure a test is needed to check that the hard coded attributes in the component template are what they're supposed to be. If they could be altered by passing options, then yes, definitely needs a test, but the component doesn't allow that.

Reply:
Removed.
But arguably, if that test isn't needed then the first tests isn't, either, since it also tests static content.
It won't matter on further iterations when the text is passed as an option anyway,